python/cli/prelude_cli/views/partner.py

```diff
@@ -81,3 +81,14 @@ def partner_deploy(controller, partner, host_ids):
     with Spinner(description='Deploying probes to hosts'):
         data = controller.deploy(partner_code=Control[partner.upper()].value, host_ids=host_ids)
     print_json(data=data)
+
+
+@partner.command('webhook-auth')
+@click.argument('partner', type=click.Choice([c.name for c in Control if c != Control.INVALID], case_sensitive=False))
```

Comment:
should this be restricted to EDR controls instead? or do you think doesn't matter/too niche?

Reply:
Yeah that's good point - I think it's worth adding the check. I'll do that